### PR TITLE
Typos in chapters 8 and 9

### PR DESCRIPTION
**Pull request by vitalydolgov**
I've gathered some typos I found while I was doing the course. I feel a bit anxious because I'm certainly not an expert in the topic... But anyway, I hope at least something is of benefit :)

### CODE REVIEW

File: src/chapters/ds/hash_tables.md [hunk 1]
```diff
@@ -547,11 +547,11 @@ module HashMap : TableMap = struct
       likely land in different buckets.
       Efficiency: O(n), where n is the number of bindings. *)
   let rehash tab new_capacity =
-    (* insert (k, v) into tab *)
+    (* insert (k, v) into [tab] *)
```
Comment (jnfoster) [inline src/chapters/ds/hash_tables.md:550]: Shouldn't the pair also be in brackets?

File: src/chapters/ds/memoization.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ convert to a single argument function).
 
 ```{code-cell} ocaml
 let memo f =
-  let h = Hashtbl.create 11 in
+  let h = Hashtbl.create 16 in
```
Comment (jnfoster) [inline src/chapters/ds/memoization.md:106]: Why?

**Review by jnfoster — CHANGES_REQUESTED** [review]
Looks good, but can you address the two minor comments?

File: src/chapters/ds/memoization.md (hunk 2 above)
Comment (vitalydolgov) [reply src/chapters/ds/memoization.md:106]: I changed the initial size for it to be consistent with other examples from "Hash Tables" chapter, and even in this chapter just the next snippet defines a table of size 16. Also, IIRC prof. Clarkson mentioned somewhere in the videos, that a power of 2 is the optimal size for performance of a hash table as it doubles and halves all the time, but I can be wrong.

File: src/chapters/ds/hash_tables.md (hunk 1 above)
Comment (vitalydolgov) [reply src/chapters/ds/hash_tables.md]: I agree, standard library also emphasizes such things.

File: src/chapters/ds/memoization.md (hunk 2 above)
Comment (jnfoster) [reply src/chapters/ds/memoization.md:106]: I haven't watched all the videos. But, depending on the hash function being used, having the capacity of the underlying structure be a power of 2 can be *anti*-optimal. So often a prime number is used as the initial value instead. @clarksmr can you recall what you said in the videos?

Comment (clarksmr) [reply src/chapters/ds/memoization.md:106]: If I did say something like that, it was most certainly an error.  I wonder if OP is thinking about the amortized analysis:  I use power-of-two sizes there to make the analysis easier.  Or maybe I pointed out that the default minimum number of buckets in `Hashtbl` [is 16](https://github.com/ocaml/ocaml/blob/53d5d05daa1f3046344a54c9d656402a00736872/stdlib/hashtbl.ml#L74).

Comment (jnfoster) [reply src/chapters/ds/memoization.md:106]: Thanks. And sorry to bother you on a Sunday morning!

Comment (vitalydolgov) [reply src/chapters/ds/memoization.md:106]: I'm sorry I was wrong :( @clarksmr certainly didn't say that in videos, the thing I was referenced to is [this phrase](https://github.com/cs3110/textbook/blob/7b22e76b32c244583a58d27ee8431669dbad8974/src/chapters/ds/hash_tables.md?plain=1#L694) regarding efficiency of a hash function, which I interpreted incorrectly.

>  So often a prime number is used as the initial value instead.

To me it's not obvious at all, could you please consider adding such important knowledge to lectures?

I will change this "fix" back, let me know if you want to update other `Hashtbl.create`. Thank you for your patience.

**Comment by vitalydolgov** [comment]
Hi there! Are changes in this PR still valid and waiting for something to happen before merge, or should I just close it?

**Pull request merged**